blog/docs/articles/getting-started-blog-p1.md

```diff
@@ -0,0 +1,262 @@
+
+When you feel like you have made some progress or that your changes are done, be sure to make a commit. For more information on making commits, read [here](https://www.atlassian.com/git/tutorials/saving-changes/git-commit){:target="_blank"}.
+Git commits need a message, so try and provide something as this will make it easier for others to understand your changes. For example, "worked on bug fix" __would not be a great
+commit message__, while "added try catch block to prevent crash" would be better as it __describes what you changed__.
+
+### Making a PR/MR
+
+![](/blog/images/getting-started-blog-series/post1/ready-for-review.png)
+
+There are two scenarios where you would likely want to open a PR (or a Merge Request (MR) if your project uses GitLab). The first is that you feel your changes are complete and 
+are ready to be merged into the project. The second is that you have made some progress but are not finished yet, and would like to get some early feedback or assistance on your changes.
+In both cases, you will want to go through the basic flow to create a PR or MR. If you are using GitHub and are unsure of how to do this, you can follow the instructions
+[here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork){:target="_blank"}.
+If you are using GitLab and are unsure of how to do this, you can follow the instructions [here](https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html#merge-changes-back-upstream){:target="_blank"}.
+
+When creating your PR/MR it is important to provide the context of why you are making the changes, what you changes, and how people can test your changes. It is good practice to 
+include a link to the issue the PR/MR is addressing, and to give a bried description of the changes you made. Additionally, your PR/MR should have a brief and descriptive title
+to make it clearer to reviewers what it contains.
+
+If your changes are incomplete, you will want to mark the PR/MR as a __draft__, to indicate to reviewers that your work is not complete. Instructions for Github are 
+[here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft){:target="_blank"}, and instructions for Gitlab are [here](https://docs.gitlab.com/ee/user/project/merge_requests/drafts.html#mark-merge-requests-as-drafts){:target="_blank"}. You may also
+want to title your PR/MR as "[WIP]: \<descriptive title>". If you have any questions about your changes, this draft PR/MR is a great to command asn ask them as whoever answers
+you will have the context of everything you have done so far. Once you feel your changes are complete/ready, you can convert your PR/MR from its draft status to being ready.
```

Comment:
`If you have any questions about your changes, this draft PR/MR is a great to command asn ask them as whoever answers you will have the context of everything you have done so far.`
Not sure, what you want to say here

Reply:
Hi @Leo6Leo,
thanks for your reply. My issue is mostly with the wording this part `... this draft PR/MR is a great to command asn ask them as whoever answers...` (or does "asn" stand for something specific?)